packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -347,14 +348,21 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
     defaultRender?: (props?: IPanelProps) => JSX.Element | null,
     headerTextId?: string | undefined
   ): JSX.Element | null => {
-    const { headerText } = props;
+    const { headerText, headerTextProps = {} } = props;
+    const headerTextNativeProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(headerTextProps, divProperties, ['id', 'className']);
 
     if (headerText) {
       return (
         <div className={this._classNames.header}>
-          <p className={this._classNames.headerText} id={headerTextId} role="heading" aria-level={2}>
+          <div
```

Comment:
Why the switch to `div`? Could this cause subtle styling regressions?

Reply:
it's not semantically correct to use `p` here. Since i am adding props to spread on this element, I decided to change the element type here so i don't need to type it to be `React.AllHTMLAttributes<HTMLParagraphElement>`.

There is no visual regression from this change. But there is down side - if user is styling it using element type selector, this change is breaking. thoughts?

Reply:
The only hesitation I have is that this is actually an element with a styles classname. I'm not sure how practical a concern it is, but most of my fear would come from exposure. How many components does `DialogContent` get reused in? I'm not sure...

Reply:
The stat report Ken made: https://msit.powerbi.com/groups/cbe7890b-e262-4c36-97c2-d6efc21c2373/reports/0747523d-9ce3-4676-8525-d9673e54cab6/ReportSection?ctid=72f988bf-86f1-41af-91ab-2d7cd011db47

In general , the usage of DialogContentStyles prop is really low. also, since we have 2 better ways for user to target the style of the title:
1) Style prop
2) Static class name
The chance that user is targeting the element using its type is quite low.

Overall, I still lean towards updating the element type here